pkg/volume/util/util.go

```diff
@@ -192,3 +193,38 @@ func CheckNodeAffinity(pv *v1.PersistentVolume, nodeLabels map[string]string) er
 	}
 	return nil
 }
+
+// ValidateLocalPath validates a single path, returning an error if the path is
+// invalid. Paths may not:
+//
+// 1. be relative
+// 2. contain '..' as an element
+// 3. start with '..'
+// 4. contain filename larger than 255 characters
+// 5. be longer than 4096 characters
+func ValidateLocalPath(targetPath string) error {
+	if targetPath == "" {
+		return fmt.Errorf("invalid path: must not be empty: %q", targetPath)
```

Comment:
1. if it's empty then I don't see a reason of showing it in quotes
2. I don't think that we should add "invalid path:" prefix to each error message. We're using `field.Invalid()` and it also adds similar prefix to the error AFAIR.